docs/api/modules/testing.md

```diff
@@ -2,8 +2,63 @@
 
 ## `createTestContext`
 
-todo
+Setup a test context providing utilities to query against your GraphQL API.
+
+This context can be augmented by plugins.
+
+#### Signature
+
+```ts
+function createTestContext(opts?: CreateTestContextOptions): Promise<TestContext>
+```
+
+#### Examples
+
+With Jest
```

Comment:
```suggestion
##### Example With Jest
```

Reply:
We currently don't have a heading level for examples because nesting/density ux issues. Some places there are 3+ examples and it becomes useful to have each one linkable, which requires titling. The change is just consistency.